orca-interlink/orca-interlink.gradle

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This module seems more like a `orca-interlink-aws` ... ?

Someone could come along and re-implement the pub/sub bits against something other than `aws` right?

Reply:
yes, I was thinking the other pubsub systems could be handled in the same module. Do you think it's worth it to carve it out in `orca-interlink-aws`?

Reply:
it seems like there is very little aws-y things in here, just the config or am I missing something if that's the case i think prob easier not to make another module?